collectors/charts.d.plugin/sensors/metadata.yaml

```diff
@@ -44,7 +44,14 @@ modules:
           description: ""
     setup:
       prerequisites:
-        list: []
+        list:
+          - title: "Install charts.d plugin and enable the sensors collector"
+            description: |
+              If using our official native DEB/RPM packages, make sure `netdata-plugin-chartsd` is installed.
+
+              The `sensors` collector is disabled by default.
+
+              To enable the collector, you need to edit the configuration file of `charts.d/sensors.conf` in `/etc/netdata`. You can do so by using the `edit-config` script. Uncomment the `sensors`, option and set the value to `force`.
```

Comment:
I think enabling the plugin should probably be a separate prerequisite entry from installing it if we’re going to have it as a prerequisite.

Essentially, this is a distinct, unrelated step from installing the plugin, so IMO it makes more sense for it to be a separate entry.

Reply:
I would also raise a point about mentioning `/etc/netdata` and not something like 
```
... the
Netdata [config directory](https://github.com/netdata/netdata/blob/master/docs/configure/nodes.md)

```

What do you think? :thinking:

Reply:
I've changed the wording on how to enable the collector, and copied what we did yesterday on e.g. https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/go_expvar/metadata.yaml

I kinda agree we could move the installing the charts.d plugin to another section, but where should it go?

Reply:
@MrZammler I agree with @Ferroin - we need to split installing/enabling. Let's make it 2 items (prereq is a list).

Reply:
Done!